config/database.yml

```diff
@@ -5,6 +5,8 @@ development:
   adapter: postgresql
   database: theodinproject
   host: localhost
+  username: 
```

Comment:
Looks like theres the database.yml is still in the PR @leosoaivan 😛 

You can get it out of the pr by checking out the master version of the file in your branch locally:
`git checkout master config/database.yml`

and then add and make a new commit